core/src/cell.rs

```diff
@@ -56,27 +59,32 @@ impl CellMetaBuilder {
             out_point,
             block_info,
             cellbase,
-            capacity,
-            data_hash,
+            data_bytes,
+            mem_cell_data,
         } = cell_meta;
         Self {
             cell_output,
             out_point,
             block_info,
             cellbase,
-            capacity,
-            data_hash,
+            data_bytes,
+            mem_cell_data,
         }
     }
 
-    pub fn from_cell_output(cell_output: CellOutput) -> Self {
+    pub fn from_cell_output(cell_output: CellOutput, data_bytes: u32) -> Self {
```

Comment:
Refactroing this fn and remove other builder fns: `cell_output` / `mem_cell_data` / `data_bytes `  

```suggestion
    pub fn from_cell_output(cell_output: CellOutput, data: Bytes) -> Self {
        let mut builder = CellMetaBuilder::default();
        builder.cell_output = cell_output;
        builder.data_len = data.len();
        builder.mem_cell_data = Some(data);
        builder
    }
```

Reply:
Pls see the comment. https://github.com/nervosnetwork/ckb/pull/1279/files#diff-cd1aaf797129c918472feb1228be7188R38